fluentui_listitem/src/main/res/values/attrs_list_sub_header.xml

```diff
@@ -5,13 +5,13 @@
   -->
 
 <resources>
-    <declare-styleable name="ListSubHeaderView">
-        <attr name="title" />
+    <declare-styleable name="FluentUIListSubHeaderView">
+        <attr name="fluentuiListItemView_title" />
         <attr name="titleColor" format="enum">
```

Comment:
missed this? Please rename this too

Reply:
updated to "fluentuiListSubHeaderView_titleColor" in last commit